helm/polaris/build.gradle.kts

```diff
@@ -0,0 +1,284 @@
+
+    val outputFile = helmTestReportsDir.get().file("helm-unit/test.log").asFile
+
+    doFirst {
+      outputFile.parentFile.mkdirs()
+      val outStream = outputFile.outputStream()
+      standardOutput = outStream
+      errorOutput = outStream
+    }
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+          echo "====== Install helm-unittest plugin ======"
+          helm plugin install https://github.com/helm-unittest/helm-unittest.git || true
+
+          echo "====== Run helm unit tests ======"
+          helm unittest .
+        """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "templates/**/*", "tests/**/*") }
+    )
+
+    outputs.cacheIf { true }
+  }
+
+val chartTestingLint by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run chart-testing (lint)"
+
+    val outputFile = helmTestReportsDir.get().file("ct/lint.log").asFile
+
+    doFirst {
+      outputFile.parentFile.mkdirs()
+      val outStream = outputFile.outputStream()
+      standardOutput = outStream
+      errorOutput = outStream
+    }
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+          ct lint --debug --charts .
+        """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "ci/*.yaml", "templates/**/*") }
+    )
+
+    outputs.cacheIf { true }
+  }
+
+// Task to build Docker images in minikube's Docker environment
+val buildMinikubeImages by
+  tasks.registering(Exec::class) {
+    group = "build"
+    description = "Build Polaris Docker images in minikube's Docker environment"
+
+    workingDir = rootProject.projectDir
+
+    val outputFile = helmTestReportsDir.get().file("minikube-images/build.log").asFile
+    val digestFile = helmTestReportsDir.get().file("minikube-images/build.sha256").asFile
+
+    doFirst {
+      outputFile.parentFile.mkdirs()
+      digestFile.parentFile.mkdirs()
+      val outStream = outputFile.outputStream()
+      standardOutput = outStream
+      errorOutput = outStream
+    }
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    echo "====== Check if minikube is running ======"
+    if ! minikube status >/dev/null 2>&1; then
+      echo "Minikube is not running. Starting minikube..."
+      minikube start
+    fi
+
+    echo "====== Set up docker environment and build images ======"
+    eval $(minikube -p minikube docker-env)
+
+    echo "====== Build server image ======"
+    docker build -t apache/polaris:latest \
+      -f runtime/server/src/main/docker/Dockerfile.jvm \
+      runtime/server
+
+    {
+      docker inspect --format='{{.Id}}' apache/polaris:latest
+    } > ${digestFile.relativeTo(workingDir)}
+  """
+          .trimIndent(),
+      )
+
+    dependsOn(":polaris-server:quarkusBuild")
+
+    inputs.files(runtimeServerDistribution)
```

Comment:
@snazy I think this is the only task that actually needs this configuration as input.

As for the Dockerfile, I don't see any other way to reference it as it is not an output of any task (that I know of). Should I create a configuration in `polaris-server` to expose it?